config/resources/defaults.yaml

```
@@ -17,6 +17,15 @@
 Debug: false
 Logging:
   Level: "Error"
+  XrootdScitokensTrace: all
+  XrootdPssTrace: all
+  XrootdCmsTrace: all
+  XrootdOfsTrace: all
+  XrootdPfcTrace: info
+  XrootdXrdTrace: all -sched
+  XrootdXrootdTrace: emsg login stall redirect
+  XrootdPssSetOptCache: DebugLevel 3
+  XrootdPssSetOptOrigin: DebugLevel 1
```

The defaults are way too verbose and impact performance. Here are the defaults OSG ships with:

```suggestion
  XrootdScitokensTrace: none
  XrootdPssTrace: off
  XrootdCmsTrace: -all
  XrootdOfsTrace: -all
  XrootdPfcTrace: info
  XrootdXrdTrace: -all
  XrootdXrootdTrace: emsg login stall redirect
  XrootdPssSetOptCache: DebugLevel 1
  XrootdPssSetOptOrigin: DebugLevel 1
```

We tell people that if they want to debug they should set
```
ofs.trace all
xrd.trace all
cms.trace all
scitokens.trace all
pss.setopt DebugLevel 3
```